### PR TITLE
getent: fix group - name to gid calculation

### PR DESCRIPTION
**Pull request by furiel**
Fixes: https://github.com/balabit/syslog-ng/issues/1757

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by furiel** [comment]
@kira-syslogng retest this please test branch=furiel-getent-group;

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: furiel-getent-group and test suite: functions

**Comment by furiel** [comment]
@kira-syslogng retest this please test branch=furiel-getent-group;

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: furiel-getent-group and test suite: functions

File: modules/getent/tfgetent.c [hunk 1]
```diff
@@ -77,6 +77,13 @@ _getent_format_array(gchar *name_or_gid, gpointer members, GString *result)
   return TRUE;
 }
 
+static gboolean
+_getent_format_array_at_location(gchar *name_or_gid, gpointer location, GString *result)
```
Comment (Kokan) [inline modules/getent/tfgetent.c]: Why the new function is needed ?
The original `_getent_format_array` is not used anywhere else.

**Pull request merged**